cli/cmd/repair.go

```diff
@@ -90,7 +91,17 @@ linkerd upgrade.`,
 			}
 
 			// Render
-			overrides, err := renderOverrides(&values, controlPlaneNamespace, true)
+			var valuesMap map[string]interface{}
```

Comment:
Can you explain this change? Why do we marshal and unmarshal the values?

Reply:
`renderOverrides` now takes a `chartutil.Values` i.e `map[string]interface{}` and hence we had to do a conversion from `Values` to `map`[string]interface{}` using Marshal and Unmarshal. 🤔

Reply:
what do you think of moving this into a function on `linkerd2.Values` called `toChartValues` which returns a `chartutil.Values`?  I think that would help the logic in repair.go and in install.go easier to understand.

Reply:
Okay added a `ToMap` func to `Values` and using it in `repair` now. Its not useful in `install` as `chartutil.CoalesceValues` gives a Map directly there.